pkg/handler/common/provider/eks.go

```diff
@@ -47,13 +47,11 @@ func listEKSClusters(cred EKSCredential, region string) ([]*string, error) {
 		return nil, err
 	}
 
-	req, res := client.EKS.ListClustersRequest(&eks.ListClustersInput{})
-	err = req.Send()
+	clusters, err := eksprovider.ListCluster(client)
 	if err != nil {
-		return nil, fmt.Errorf("cannot list clusters in region=%s: %w", region, err)
+		return nil, err
 	}
-
-	return res.Clusters, nil
+	return clusters, nil
```

Comment:
If you're not adding error context, this can just be `return eksprovider.ListCluster(client)`.

Also, what is "ListCluster"? Does it list a single cluster? Should it maybe be called "ListClusters"?